### PR TITLE
Segment analytics for CRT

### PR DESCRIPTION
**Pull request by attemka**
#6001 

few things to note @dmtrjsg:

1. Page visit will show 'CRT Dashboard', selected tab will be in params. Same as we did on YPP Dashboard.
2. Can't track 'Tokens sold on AMM' as this can happen while user is away and we're only tracking user events. Although, 'Tokens purchased on AMM' is tracked and should cover it.

### CODE REVIEW

File: packages/atlas/src/components/_crt/BuyMarketTokenModal/BuyMarketTokenModal.tsx [hunk 1]
```diff
@@ -164,6 +167,12 @@ export const BuyMarketTokenModal = ({ tokenId, onClose: _onClose, show }: BuySal
         ),
       onTxSync: async () => {
         if (memberTokenAccount?.tokenAccounts.length) {
+          trackAMMTokensPurchased(
+            data?.creatorTokenById?.symbol ?? 'N/A',
```
Comment (WRadoslaw) [inline packages/atlas/src/components/_crt/BuyMarketTokenModal/BuyMarketTokenModal.tsx:172]: We shouldn't use symbol as it is not unique

File: packages/atlas/src/components/_crt/CloseMarketModal/CloseMarketModal.tsx [hunk 2]
```diff
@@ -76,6 +78,7 @@ export const CloseMarketModal = ({ onClose, show, channelId, tokenId }: CloseMar
         txFactory: async (updateStatus) =>
           (await joystream.extrinsics).closeAmm(memberId, channelId, proxyCallback(updateStatus)),
         onTxSync: async () => {
+          trackAMMClosed(symbol ?? 'N/A', channelId)
```
Comment (WRadoslaw) [inline packages/atlas/src/components/_crt/CloseMarketModal/CloseMarketModal.tsx]: Same as above

**Review by WRadoslaw — CHANGES_REQUESTED** [review]

**Review by WRadoslaw — APPROVED** [review]
lgtm

**Comment by dmtrjsg** [comment]
> Tokens sold on AMM' as this can happen while user is away and we're only tracking user events. Although, 'Tokens purchased on AMM' is tracked and should cover it.

@attemka user can initiate sale via GUI, that's what I was asking for here. I hope that was implemented? Its the same operation as purchased but user just clicks "sell" instead of "purchase".

**Review by dmtrjsg — APPROVED** [review]
ok

**Comment by WRadoslaw** [comment]
Looks good

**Pull request merged**